translator/config/schema.json

```diff
@@ -1113,6 +1113,10 @@
         "region_override": {
           "description": "The override region",
           "type": "string"
+        },
+        "transit_spans_in_otlp_format": {
```

Comment:
Did we settle on including "spans" in the field name?

Reply:
yes we had some discussion internally within X-Ray and this is the config name we want. But do let me know if there are concerns from cwagent side.